### PR TITLE
fix: now list activity graphs only load data client-side

### LINKED ISSUE #2213: Bug: Lists Activity error when trying to load large number of contributors
### Describe the bug

Something is causing an error when we try to load the list activity for large lists.

It seems like I can load a list of 60, but not at 94.

image
 

### Steps to reproduce

1. Go to https://app.opensauced.pizza/lists/7eef76df-a63a-42b8-babd-237b9b213579/activity
2. See a 502

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Now only list info is loaded on the list activity page. All graph data is pulled on the client-side only now.

Here's a public list loading data for graphs client-side only now, https://deploy-preview-2215--oss-insights.netlify.app/lists/8b371419-6219-4448-9de1-f6293a25f98e/activity?range=30

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2213

## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/app/assets/833231/75e7e0ae-3c7f-47f1-bff0-98b9743d06f8

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: pages/lists/[listId]/activity.tsx [hunk 1]
```diff
@@ -25,11 +22,6 @@ interface ContributorListPageProps {
   list?: DBList;
   numberOfContributors: number;
   isError: boolean;
-  activityData: {
```
Comment (nickytonline) [inline pages/lists/[listId]/activity.tsx:28]: All initial graph data from the server-side has been removed so only essential list information is loaded from the server-side.

File: pages/lists/[listId]/activity.tsx [hunk 2]
```diff
@@ -167,17 +131,14 @@ const ListActivityPage = ({
         <Error errorMessage="Unable to load list activity" />
       ) : (
         <div className="grid grid-cols-1 xl:grid-cols-2 gap-4">
-          <ClientOnly>
```
Comment (nickytonline) [inline pages/lists/[listId]/activity.tsx:170]: I removed the client-only component as it's not required anymore as data only loads client-side now for graphs.

**Review by OgDev-01 — APPROVED** [review]
🍕

**Comment by nickytonline** [comment]
I confirmed with @isabensusan that for her 200 contributors list, it's all good now.

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**